app/controllers/concerns/idv_session.rb

```diff
@@ -1,129 +1,29 @@
 module IdvSession
   extend ActiveSupport::Concern
 
-  if Figaro.env.proofing_vendors
-    Figaro.env.proofing_vendors.split(/\W+/).each do |vendor|
-      vendor_path = "#{Rails.root}/vendor/#{vendor}/lib"
-      $LOAD_PATH.unshift vendor_path
-    end
-  end
-
   def confirm_idv_session_started
-    redirect_to idv_session_url unless idv_session[:params].present?
+    redirect_to idv_session_url unless idv_session.params.present?
   end
 
   def confirm_idv_attempts_allowed
-    if idv_attempter.exceeded?
+    attempter = idv_session.attempter
```

Comment:
Any particular reason to go through the `Idv::Session` class to create an instance of `Idv::Attempter`, as opposed to calling `Idv::Attempter` directly?